scripts/integration.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
To properly test this, it should be enabled on the github actions, here:

https://github.com/aiven/karapace/blob/eb5a7720475b66de5011722db4799e3815fdacaa/.github/workflows/tests.yml#L41-L43